File: 08 Rails/bacmd/app/models/reservation.rb
# == Schema Information
#
# Table name: reservations
#
#  id         :integer          not null, primary key
#  user_id    :integer
#  flight_id  :integer
#  rows       :integer
#  columns    :integer
#  created_at :datetime         not null
#  updated_at :datetime         not null
#

class Reservation < ActiveRecord::Base
  belongs_to :flight
  belongs_to :user
end

File: 11_backbone/burningAirlinesFinal/app/assets/javascripts/models/Reservation.js
var app = app || {};

app.Reservation = Backbone.Model.extend({
  urlRoot: "/reservations",
  default: {
    user_id: "",
    flight_id : "",
    columns: "",
    rows: "",
  },
  initialize: function() {
    console.log("a new reservation was create")
  }
})
